### DISCUSSION TITLE
openai 向量化的时候报错 :Query was not successful! Unexpected status code: 422, with response body: {'error': [{'message': 'no graphql provider present, this is most likely because no schema is present. Import a schema first!'}]}.; [openai] Error: argument sHelp #6698Help

### DISCUSSION THREAD

**Discussion by shanaozi** [langgenius/dify#6703] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

openai 向量化的时候报错 :Query was not successful! Unexpected status code: 422, with response body: {'error': [{'message': 'no graphql provider present, this is most likely because no schema is present. Import a schema first!'}]}.; [openai] Error: argument sHelp #6698Help
![1721962095009](https://github.com/user-attachments/assets/8497b0ca-db92-47d8-a843-14133544983d)

**Comment by yinziqiong** [comment]
请问解决了吗
